lib/api/redfish-1.0/systems.js

```diff
@@ -1918,19 +1916,22 @@ var addVolume = controller(function(req,res) {
                 switch(raidLevel)
                 {
                     case "NonRedundant":
-                        graphOptions.defaults.raidLevel = 'r0';
+                        graphOptions.defaults.raidLevel = 0;
                         break;
                     case "Mirrored":
-                        graphOptions.defaults.raidLevel = 'r1';
+                        graphOptions.defaults.raidLevel = 1;
                         break;
                     case "StripedWithParity":
-                        graphOptions.defaults.raidLevel = 'r5';
+                        graphOptions.defaults.raidLevel = 5;
+                        break;
+                    case "RawDevice":
+                        graphOptions.defaults.raidLevel = 6;
```

Comment:
RawDevice means that `The volume is a raw physical device without any RAID or other virtualization applied.`  https://github.com/RackHD/on-http/blob/master/static/DSP8010_2016.3/metadata/Volume_v1.xml#L180-L182
Please remove RawDevice here.

Reply:
ok